components/org.wso2.carbon.identity.application.authenticator.push.device.handler/src/test/java/org/wso2/carbon/identity/application/authenticator/push/device/handler/PushDeviceHandlerTest.java

```diff
@@ -132,7 +131,7 @@ public void mockGetDevice() throws Exception {
     public void mockRegistrationCacheEntry() throws Exception {
         cache = spy(RegistrationRequestChallengeCache.getInstance());
         RegistrationRequestChallengeCacheEntry entry = new RegistrationRequestChallengeCacheEntry(UUID.randomUUID(),
-                "avishka", "PRIMARY", "carbon.super", false);
+                "avishka", "carbon.super", false);
```

Comment:
have a generic name instead

Reply:
Tracked by #71. Will be fixed in a separate PR for tests